### ISSUE TITLE
Issue with global_step for single GPU

### ISSUE THREAD

**Issue by shyam-jagannathan** [Zehaos/MobileNet#56]
I have a single Nvidia 1070 GPU, running TF 1.3 on Ubuntu 16.04 and Python 2.

When I try to train mobilenet from scratch on the "Flowers" dataset, i see that the global_step does not increment at all. I tried looking for workaround and found this, https://github.com/tensorflow/tensorflow/issues/8190

But even after explicitly declaring global_step, i see that its stuck at 0. Does the training not work for 1GPU(num_clones=1)?

INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:global step 0: loss = 1.9280 (3.353 sec/step)
INFO:tensorflow:global step 0: loss = 29.2923 (0.471 sec/step)
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 0: loss = 10.5883 (0.469 sec/step)
INFO:tensorflow:global step 0: loss = 5.5462 (0.458 sec/step)
INFO:tensorflow:global step 0: loss = 2.7223 (0.463 sec/step)
INFO:tensorflow:global step 0: loss = 8.0004 (0.456 sec/step)
INFO:tensorflow:global step 0: loss = 7.2813 (0.465 sec/step)
INFO:tensorflow:global step 0: loss = 6.9499 (0.462 sec/step)
INFO:tensorflow:global step 0: loss = 4.5250 (0.461 sec/step)
INFO:tensorflow:global step 0: loss = 1.6212 (0.462 sec/step)
INFO:tensorflow:global step 0: loss = 3.6059 (0.457 sec/step)

**Comment by zhenglaizhang** [comment]
@shyam-jagannathan Hi, I also met with the same issue, have you figured this out?
